data/crac-io/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/JsonRetrocompatibilityTest.java

```diff
@@ -249,10 +244,30 @@ void importV1Point9Test() {
         testContentOfV1Point9Crac(crac);
     }
 
+    @Test
+    void importV1Point10Test() {
+        // Add support for user-defined Instants
+        InputStream cracFile = getClass().getResourceAsStream("/retrocompatibility/v1/crac-v1.10.json");
+
+        Crac crac = new JsonImport().importCrac(cracFile);
```

Comment:
you should test the instants of the CRAC, which is the most important feature of this new version

Reply:
I tested this in the end of the file, is it ok ?

Reply:
ok sorry i forgot you put it there